ios/ConverseNotificationExtension/Xmtp/Client.swift

```diff
@@ -107,11 +107,10 @@ func putGroupInviteRequest(apiURI: String?, account: String, xmtpClient: Client,
     do {
       
       let joinRequestUri = "\(apiURI)/api/groupJoinRequest/\(joinRequestId)"
-      let privateKey = try PrivateKey(xmtpClient.keys.identityKey)
       guard let digest = "XMTP_IDENTITY".data(using: .utf8) else {
         return
       }
-      let signature = try await privateKey.sign(digest).serializedData().base64EncodedString()
+      let signature = try await xmtpClient.keys.identityKey.sign(digest).serializedData().base64EncodedString()
```

Comment:
Apparently iOS is still not verifying in Javascript. The Android and iOS code is identical and I can validate in a test that the bytes being passed are the same. So there has to be something weird going on here. I'll keep looking into this when I get time.